validator/client/runner.go

```diff
@@ -121,7 +59,8 @@ func run(ctx context.Context, v iface.Validator) {
 		log.Fatalf("Failed to update proposer settings: %v", err) // allow fatal. skipcq
 	}
 	for {
-		slotCtx, cancel := context.WithCancel(ctx)
+		var slotCtx context.Context
```

Comment:
why is this separated out, this will break our validator's deadline per slot.

Reply:
I changed it again. Defining `slotCtx` here is actually not necessary as it will be reassigned later before use. Does it look good to you? I don't think there will be semantic diff now.